src/BursarExportPlugin.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Button,
+  LoadingPane,
+  Pane,
+  PaneFooter,
+} from '@folio/stripes/components';
+import { useStripes } from '@folio/stripes/core';
+import { FormApi } from 'final-form';
+import React, { useCallback, useRef } from 'react';
+import { FormattedMessage } from 'react-intl';
+import formValuesToDto from './api/dto/to/formValuesToDto';
+import schedulingToDto from './api/dto/to/schedulingToDto';
+import useAutomaticSchedulerMutation from './api/mutators/useAutomaticSchedulerMutation';
+import useManualSchedulerMutation from './api/mutators/useManualSchedulerMutation';
+import ConfigurationForm, { FORM_ID } from './form/ConfigurationForm';
+import useInitialValues from './hooks/useInitialValues';
+import FormValues from './types/FormValues';
+
+export default function BursarExportPlugin() {
+  const stripes = useStripes();
+
+  const scheduleManually = useManualSchedulerMutation();
+  const scheduleAutomatically = useAutomaticSchedulerMutation();
+
+  const formApiRef = useRef<FormApi<FormValues>>(null);
+
+  const submitCallback = useCallback(async (values: FormValues) => {
+    if (values.buttonClicked === 'manual') {
+      await scheduleManually(formValuesToDto(values));
+    } else {
+      await scheduleAutomatically({
+        bursar: formValuesToDto(values),
+        scheduling: schedulingToDto(values.scheduling),
+      });
+    }
+  }, []);
+
+  const initialValues = useInitialValues();
+
+  const handleRunManuallyClick = useCallback(() => {
+    formApiRef.current?.change('buttonClicked', 'manual');
+  }, []);
+
+  const handleSaveClick = useCallback(() => {
+    formApiRef.current?.change('buttonClicked', 'save');
+  }, []);
+
+  if (initialValues === null) {
+    return (
+      <LoadingPane
+        paneTitle={
+          <FormattedMessage id="ui-plugin-bursar-export.bursarExports.paneTitle" />
+        }
+        defaultWidth="fill"
+        footer={
+          <PaneFooter
+            renderStart={
+              <Button disabled>
+                <FormattedMessage id="ui-plugin-bursar-export.bursarExports.button.runManually" />
+              </Button>
+            }
+            renderEnd={
+              <Button disabled buttonStyle="primary">
+                <FormattedMessage id="ui-plugin-bursar-export.bursarExports.button.save" />
+              </Button>
+            }
+          />
+        }
+      />
+    );
+  }
+
+  return (
+    <Pane
+      defaultWidth="fill"
+      footer={
+        <PaneFooter
+          renderStart={
+            <Button
+              disabled={!stripes.hasPerm('data-export.job.item.post')}
+              type="submit"
```

Comment:
why do you need 2 submit types? because of this you create extra functions `handleRunManuallyClick`, `handleSaveClick`, `submitCallback` + formApiRef to support it

why not to have `handleRunManually` as just a button and `saveSmth` on submit?

Reply:
This was a direct replication of [the current behavior](https://folio-snapshot.dev.folio.org/settings/users/transfer-criteria), which had "run manually" as a separate option from the saved/scheduled configuration (also may be beneficial to run one as a one-off, before overwriting the current configuration)

Reply:
my question is not about 2 buttons, my questions is about 2 buttons with type="submit" and way how it's implemented

Reply:
Which button is clicked is stored as part of the form, so it can be handled by the normal submit logic.

Reply:
what do you want to achieve with it? now I see extra logic to support something, usually form supports one submit action, in this case it's `Save`, `Run manually` is not a form submit